dart/lib/src/event_processor/deduplication_event_processor.dart

```diff
@@ -63,7 +63,7 @@ class DeduplicationEventProcessor extends EventProcessor {
     // No duplication detected
     _exceptionToDeduplicate.add(exceptionHashCode);
     if (_exceptionToDeduplicate.length > _options.maxDeduplicationItems) {
-      _exceptionToDeduplicate.removeLast();
+      _exceptionToDeduplicate.removeFirst();
```

Comment:
@ueman since `add(x)` adds to the tail, we need to remove the `first` that is the oldest one, I guess its a bug, right, test only passed because the test that was doing that, didn't enable the `enableDeduplication` feature